Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -111,7 +111,7 @@ public async Task<IActionResult> Create([FromBody]Equipment equipment)
                     equipment.Space = space;
                 }
                 _context.Equipment.Add(equipment);
-                await _eventService.TrackCreateEquipment(equipment);
+                // await _eventService.TrackCreateEquipment(equipment);
```

Comment:
Do you mean to stop tracking equipment inside this date change pr?

Reply:
oops